### DISCUSSION TITLE
Marketing attribution functionality is missing

### DISCUSSION THREAD

**Discussion by njbmartin** [Shopify/storefront-api-feedback#110] [Ideas & Requests]
Pulling out the discussion here: https://github.com/Shopify/storefront-api-feedback/discussions/12#discussioncomment-1150829

This still seems to be missing from the API. With Shopify being the source of truth, especially now we've migrated to a new subscription platform that supports Shopify Checkout, it is now crucial that we understand where our customers are coming from, and being able to add UTM params into the cart would be ideal.

Can you provide some sort of timeline for this functionality?

**Comment by arobbins** [comment]
Any update on this?
